src/rmodels.c

```diff
@@ -5745,8 +5766,8 @@ static Model LoadVOX(const char *fileName)
     int verticesRemain = voxarray.vertices.used;
     int verticesMax = 65532; // 5461 voxels x 12 vertices per voxel -> 65532 (must be inf 65536)
 
-    Vector3 *pvertices = voxarray.vertices.array;        // 6*4 = 12 vertices per voxel
-    Color *pcolors = voxarray.colors.array;
+    Vector3 *pvertices = ToVector3(voxarray.vertices.array);        // 6*4 = 12 vertices per voxel
```

Comment:
Avoid those functions and do the assignment directly here.

Reply:
sure!